File: Beecrowd/Beginner/1000 - 1099/1018 - Banknotes/Python/1018_Banknotes.py
amount = int(input())
print(amount)

notes = [100, 50, 20, 10, 5, 2, 1]

for note in notes:
    noteAmount = amount // note
    
    print(f"{noteAmount} nota(s) de R$ {note},00")
    
    amount -= noteAmount * note

File: Beecrowd/Beginner/1000 - 1099/1018 - Banknotes/C++/1018_Banknotes.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int amount;
    cin >> amount;

    cout << amount << endl;

    int notes[] = {100, 50, 20, 10, 5, 2, 1};
    for (int i = 0; i < 7; i++) {
        int noteAmount = amount / notes[i];
        cout << noteAmount << " nota(s) de R$ " << notes[i] << ",00" << endl;
        amount -= noteAmount * notes[i];
    }
    
    return 0;
}